Fenced_Frames_Ads_Reporting.md

```diff
@@ -109,11 +111,21 @@ window.fence.reportEvent({
 });
 ```
 
+
+To send a request from a document that is cross-origin to the fenced frame config's mapped URL with the POST request body `''` to the URL registered for `buyer` when a user click happens:
```

Comment:
let's add a response body in this example so it doesn't look like that's not allowed